testing/testmedia.cpp

```diff
@@ -2165,7 +2165,7 @@ void SrtModel::Establish(std::string& w_name)
         if (w_name != "")
         {
             Verb() << "Connect with requesting stream [" << w_name << "]";
-            UDT::setstreamid(m_sock, w_name);
+            srt::setstreamid(m_sock, w_name);
```

Comment:
Might be better to use C API:
```c++
srt_setsockopt(m_sock, 0, SRTO_STREAMID, w_name.c_str(), w_name.size());
```

Reply:
No. These extension functions are there for a reason. The internal representation for it uses `std::string` and this function operates directly with this type. The use of `SRTO_STREAMID` is only a C-API representation for it and it uses an intermediate array of characters to pass the value.

Reply:
I am not talking about the conversion, but about mixing C and C++ API.